pilot/pkg/serviceregistry/kube/controller/multicluster.go

```diff
@@ -221,11 +211,13 @@ func (m *Multicluster) AddMemberCluster(client kubelib.Client, clusterID string)
 		}
 	}
 
-	// TODO only create namespace controller and cert patch for remote clusters (no way to tell currently)
 	if m.serviceController.Running() {
 		// if serviceController isn't running, it will start its members when it is started
+		// TODO these registries should stop on their individual stopCh to allow removing individual clusters
```

Comment:
maybe I don't understand this, but it seems like `clusterStopCh` is per-cluster? So wouldn't that allow removing individual cluster, because you would close `clusterStopCh` which is just for this cluster?

Reply:
I just need to remove a couple lines from the aggregate registry that call `Run` using the global stopCh on all child registries.